2-breakfast-query-graphql-exercise/README.md

```diff
@@ -0,0 +1,77 @@
+# Breakfast Query
+_A Revenue Engineering Exercise, for Engineer Level 2_
+
+Welcome to your first day on the job at Concert Breakfast Studio! _(in our new pivot to a breakfast commerce)._ Through extensive market research we've realized (hence the pivot) that people are fanatical about food delivery and we are fanatical about great breakfasts so we thought we could combine the two into a musical morning experience. Farm fresh goodness from our culinary experts to your hungry chomper.
+
+## The work
+
+![delicious looking interface describing food you should order](https://user-images.githubusercontent.com/848147/65905961-31605e00-e387-11e9-8404-57bd7654021d.png)
+
+Our pals [over on the client side](https://github.com/voxmedia/revenue-hiring/tree/master/2-client-side-javascript-components/README.md) have made an amazing menu and ordering interface. And now you'll be helping our patrons get their delicious breakfast order to our hard working cooks, farmers and delivery people!
+
+To get going we'll prototype this with [Apollo Server](https://www.apollographql.com/docs/apollo-server/getting-started/), which is part of our BreakfastQL sandbox we made for you, listed below.
+
+**The task is to design, and get started on, a server side system that allows** for the client interface to perform the following:
+- discover what food items are available,
+- submit an order,
+- check the status of an order,
+- see all unfilled orders, and
+- update the status of an order
```

Comment:
Is this something we'd expect to see? If so, do we want to call this out in the steps below?